composer.json

```diff
@@ -11,7 +11,6 @@
     ],
     "require": {
         "php": ">=7.4",
-        "brick/reflection": "^0.4",
```

Comment:
Is it possible to just update this to `^0.4 || ^0.5`?

Reply:
https://github.com/brick/reflection/releases/tag/0.5.0

> The following methods have been removed:
>    ReflectionTools::getParameterTypes()
>    ReflectionTools::getPropertyTypes()

with that removed, i don't know if this Package would need Brick anymore (if a Symfony one can do the same)

Reply:
Okay, cool